### ISSUE TITLE
Allow listing for system nomad jobs

### ISSUE THREAD

**Issue by thatsk** [ngine-io/chaotic#11]
right now i can see chaos is performing for all jobs but seems like chaos should happen for service nomad jobs. System jobs keep running or depend on condition how people are performing chaos. But I think we need feature where we can skip chaos testing for system jobs by adding allowlist for nomad job types.

**Comment by resmo** [comment]
Hi @thatsk, I added PR #12 addressing this topic, I would appreciate your feedback.

**Comment by thatsk** [comment]
@resmo can you provide me instruction to build the chaotic binary or docker container so that I can build container from this branch. In readme.md file I am trying to find instruction to compile it but didn't find and in docker repo there is direct coming from pip.

**Comment by thatsk** [comment]
@resmo looks like we need to add more in job type with the specific jobs.

```
  # Optional: job type skip list
  job_type_skiplist:
    - system
    - batch
    - sysbatch
  job_skiplist:
    - plugin-nfs-controller
    - prometheus-controller
```

**Comment by resmo** [comment]
@thatsk Thanks for your feedback, improving docs is on our todo list.

Another possibility we added to filter out (opt-out) is by adding a meta tag (with a key you define)

```yaml
  # Add a meta tag on in your nomad job "chaotic" = False to opt-out
  job_meta_opt_key: chaotic
  ```

**Comment by resmo** [comment]
released 0.8.0

**Comment by thatsk** [comment]
@resmo Tested in nomad cluster working as expected checked both the condition
